extension/json/src/json_scan.cpp

```diff
@@ -8,25 +12,62 @@ using namespace kuzu::common;
 namespace kuzu {
 namespace json_extension {
 
-struct JsonBindData : public TableFuncBindData {
+struct JsonScanConfig {
+    std::string format;
+    int64_t depth;
+    int64_t breadth;
+
+    explicit JsonScanConfig(const std::unordered_map<std::string, Value>& options)
+        : format("array"), depth(-1), breadth(-1) {
+        for (const auto& i : options) {
+            if (i.first == "FORMAT") {
+                if (i.second.getDataType().getLogicalTypeID() != LogicalTypeID::STRING) {
+                    throw BinderException("Format parameter must be a string!");
+                }
+                format = i.second.strVal;
```

Comment:
What will happen if the format is invalid

Reply:
fileToJson will catch this and give `RuntimeException: Invalid JSON file format: must be either 'unstructured' or 'array'`